device/iot-device-samples/android-sample/build.gradle

```diff
@@ -6,11 +6,15 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.4'
+        classpath 'com.android.tools.build:gradle:4.1.2'
```

Comment:
Out gradle versions were a bit out of date, so now seems like an appropriate time to bump them